### PR TITLE
Adding JsBarcode project

### PR DESCRIPTION
**Pull request by lindell**

### CODE REVIEW

File: files/jsbarcode/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "JsBarcode",
+  "repo": "lindell/JsBarcode",
+  "files": {
+    "include": ["bin/browser/*.min.*"]
```
Comment (jimaek) [inline files/jsbarcode/update.json]: Is this directory not required in production? https://github.com/lindell/JsBarcode/tree/v3.3.2/bin/browser/barcodes

Comment (lindell) [inline files/jsbarcode/update.json]: Not required but they should be included. Added it to the PR

File: files/jsbarcode/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "JsBarcode",
+  "repo": "lindell/JsBarcode",
+  "files": {
+    "include": ["bin/browser/*.min.*", "bin/browser/barcodes/*.min.*"]
```
Comment (jimaek) [inline files/jsbarcode/update.json]: This wont create a "barcodes" folder.
You should set the `basePath` parameter to `bin/browser/` and then do `*.min.* and `./barcodes/*.min.*`

File: files/jsbarcode/info.ini [hunk 3]
```diff
@@ -0,0 +1,5 @@
+author = "Johan Lindell"
+github = "https://github.com/lindell/JsBarcode"
+homepage = "https://github.com/lindell/JsBarcode"
+description = "JsBarcode is a barcode generator written in JavaScript. It supports multiple barcode formats and it has no dependencies but works with jQuery if you are into that."
+mainfile = "JsBarcode.min.js"
```
Comment (jimaek) [inline files/jsbarcode/info.ini]: The file is named `JsBarcode.all.min.js` here https://github.com/lindell/JsBarcode/tree/v3.3.2/bin/browser

File: files/jsbarcode/update.json (hunk 2 above)
Comment (lindell) [inline files/jsbarcode/update.json]: The bot is now saying that I should use `././barcodes/*.min.*`. That seems very unnecessary but it's maybe what I should do?

Comment (jimaek) [inline files/jsbarcode/update.json]: Ignore the bot :)

**Comment by jimaek** [comment]
Thank you

**Pull request merged**